src/net.cpp

```diff
@@ -2603,6 +2603,15 @@ int CConnman::GetBestHeight() const
     return nBestHeight.load(std::memory_order_acquire);
 }
 
+void CConnman::RelayInventory(const CInv& inv) const
+{
+    LogPrint(BCLog::NET, "Relaying inv %s\n", inv.ToString());
```

Comment:
An inv may be further not sent due to a check inside PushInventory ```filterInventoryKnown.contains```, so not sure how reasonable is this log :)

Reply:
Good point - this now only logs if the `inv` is actually to be relayed, and reports to how many peers it's being relayed.